### ISSUE TITLE
New feature: technical report

### ISSUE THREAD

**Issue by muuankarski** [UNFAOstatistics/pocketbook#2]
In addition to actual book, the process should also print out a technical report (pdf/html) page, that would have a page for each graph listing the code and the comments from the code chunk that produces the object, summary of the data and the plot/table itself. And perhaps few more items too.

Purpose for this document would be to provide answer a quick  to questions like
- what data was used
- how the growth rate was computed
- which countries are excluded
- why did you do this and not that
- could we do a ??-graph instead
- ...

**Comment by amyheyman** [comment]
Sounds like a lot of work...

**Comment by muuankarski** [comment]
No, actually it will be pretty easy! We just place the chapter into a different latex template than yearbook and it will print everything out. Just automatically once we have few spreads defined! And, I actually implemented a draft already. It is all hidden in the power of 
- [literate programming](https://en.wikipedia.org/wiki/Literate_programming)
- [knitr](http://yihui.name/knitr/)
- [knitr hooks](http://yihui.name/knitr/hooks/)
- [knitr patterns](http://yihui.name/knitr/patterns/)
